instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -151,6 +169,7 @@ def process_response(self, request, response):
             self._environ_activation_key in request.META.keys()
             and self._environ_span_key in request.META.keys()
         ):
+            self._set_http_route(request.META[self._environ_span_key], request)
```

Comment:
Would doing this in `process_request` be enough and take care of both success/error cases?

Reply:
I have only been able to find this route as being available from request.resolver_match.route and request.resolver_match is none during process_request() but become available during process_response and process_exception. So I think this value cannot be set just in process_request().